### PR TITLE
Extra info about eval command parameter

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Added a note about `--fetch-remote-resources` parameter.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3 https://github.com/uyuni-project/uyuni-docs/pull/2133
- [ ] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/20448

### CODE REVIEW

**Comment by keichwa** [comment]
@0rnela it would be good if you could add the bug number to a `.changelog` entry.

**Comment by 0rnela** [comment]
> @0rnela it would be good if you could add the bug number to a `.changelog` entry.

Done, thanks @keichwa.

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Added info about eval command parameter for OpenSCAP (bcs#1207931)
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- Added information about evaluation command parameter for OpenSCAP (bsc#1207931)
```

**Review by keichwa — APPROVED** [review]
Thank you, Ornela!

**Review by jcayouette — APPROVED** [review]

**Pull request merged**